drivers/SmartThings/zigbee-window-treatment/fingerprints.yml

```diff
@@ -92,3 +98,9 @@ zigbeeGeneric:
         - 0x0102 #Window Covering
         - 0x0001 #Power Configuration
     deviceProfileName: window-treatment-battery
+  - id: "shade-mic"
+    deviceLabel: Micronic Window Treatment (ACM301)
+    clusters:
+      server:
+        - 0x0102 #Window Covering
+    deviceProfileName: window-treatment-micronic
```

Comment:
I meant to delete this fingerprint. There is nothing here that uniquely identifies a device as being a Micronic device, so it shouldn't be labeled that way on join.